pkg/util/utils_test.go

```diff
@@ -103,10 +102,9 @@ func TestReplaceChars(t *testing.T) {
 
 func TestConcatStrings(t *testing.T) {
 	testData := []struct {
-		src         []string
-		prefix      string
-		postfix     string
-		expectedRes []string
+		src             []string
+		prefix, postfix string
```

Comment:
I feel like this hurts readability exactly in the place it's necessary the most: tests. Having one doubled up field in the struct makes the test data difficult to scan.

Reply:
fixed

Reply:
fixed